.github/workflows/build.yml

```diff
@@ -122,6 +145,7 @@ jobs:
           BUILD_ERROR_LOGFILE: /github/workspace/build_log_errors.txt
           BUILD_UNSUPPORTED_FILE: /github/workspace/build_unsupported.txt
           BUILD_SUCCESS_FILE: /github/workspace/build_success.txt
+          INCLUDE_OPTIONAL_ARCH: ${{ github.event.inputs.include_optional_arch }}
```

Comment:
just wondering whether this env variable is required.

Reply:
You're correct, it's not